src/AlgAss/AlgCyc.jl

```diff
@@ -0,0 +1,483 @@
+
+@doc raw"""
+Create a cyclic algebra.
+
+The field is expected to be a cyclic extension over its base field,
+and $sigma a generator of its galois group. This creates the algebra commonly
+denoted (K/k, σ, a). Its basis is chosen to be xᵢ⋅πʲ, where xᵢ is the basis
+of K/k, and π a generating element, i.e. whose conjugation on K acts as σ and satisfies
+πⁿ = a.
+"""
+function cyclic_algebra(
+  fld::NumField{T},
+  sigma::NumFieldHom{S, S},
+  a::T
+)::CyclicAlgebra{T, S} where {T, S<:NumField{T}}
```

Comment:
```suggestion
) where {T, S<:NumField{T}}
```
Was adding the type declaration done on purpose? We usually don't do this, since it hides bugs and adds a confusing implict `convert`.

Reply:
Makes sense, I will remove them.